experimental/execution/ptx-executor/src/state_reader.rs

```diff
@@ -45,19 +36,21 @@ impl PtxStateReader {
         scheduler: PtxSchedulerClient,
         state_view: &(impl StateView + Sync),
     ) {
-        IO_POOL.scope(move |io_scope| loop {
-            let scheduler = scheduler.clone();
-            match work_rx.recv().expect("Channel closed.") {
-                Command::Read { state_key } => io_scope.spawn(move |_io_scope| {
-                    let value = state_view.get_state_value(&state_key).unwrap();
-                    scheduler.inform_state_value((state_key, BASE_VERSION), value);
-                }),
-                Command::FinishBlock => {
-                    trace!("finish_block.");
-                    break;
-                },
-            }
-        });
+        THREAD_MANAGER
+            .get_exe_cpu_pool()
```

Comment:
Looks evil :trollface:

Reply:
switched to high_pri_io_pool 

we are evil 🐶